x-pack/plugins/security_solution/public/management/components/console/components/command_usage.tsx

```diff
@@ -45,7 +45,7 @@ export const CommandInputUsage = memo<Pick<CommandUsageProps, 'commandDef'>>(({
                 })}
               </ConsoleCodeBlock>
             ),
-            description: usageHelp,
+            description: usageHelp && usageHelp.length > 0 ? usageHelp : commandDef.name,
```

Comment:
I would have assumed that `getCommandNameWithArgs()` (in `console/service/utils`) would have been used instead.  I'm going to 👍 this, but we should think about some refactoring - see below.

------

So it looks like to me that we now have multiple methods attempting to build command usage - there should be only one method.

I lost track of past changes, but it looks like `getArgumentsForCommand()` (used in the `useMemo()` above) was changed at some point to return an array - I think it was when exclusiveOR was introduced. It seems to be returning Usage examples now _(which is what the command definition `usageExample` was introduced for)_ where before it just returned a string with the arguments supported by the command.  That method should probably now be renamed to `getArgumentUsageForCommand()` or maybe we should just have 1 utility/service function that generates command usage(s).

btw - side note: I noticed this method in `console/services/parse_command_input` - `parsedPidOrEntityIdParameter()`. This method should be moved out of `console/` and into the `endpoint_responder` folder somewhere.